foundations/project-etch-a-sketch/js/script.js

```diff
@@ -0,0 +1,66 @@
+const GAME_BOARD = document.querySelector('#grid-container');
+const GRID_SIZE_BTN = document.querySelector('#change-grid');
+const GRID_MAX_SIZE = 512;
+const CELL_THRESHOLDS = [1, 100];
+const RGB_MAX_VALUE = 255;
+
+const Grid = class {
+    constructor(size) {
+        this.width = size || 16;
+        this.height = this.width;
+        this.totalCells = this.width ** 2;
+        this.cellSize = `${GRID_MAX_SIZE/this.width}px`;
+    }
+
+    isValidGridSize(num,min,max) {
```

Comment:
```suggestion
    isValidGridSize(num, min, max) {
```

Convention is to put a space between function params.